shared/src/ledger/protocol/mod.rs

```diff
@@ -245,6 +252,10 @@ where
             );
             Ok(TxResult::default())
         }
+        ProtocolTxType::BridgePool(vexts) => {
+            transactions::bridge_pool_roots::apply_derived_tx(storage, vexts)
+                .map_err(Error::ProtocolTxError)
+        }
```

Comment:
Since the code works for both digests and single vext, I went ahead an wrote the code. It won't pass process proposal without the feature flag though

Reply:
Yeah we shouldn't expect digest transactions at all (e.g. `BridgePool`) until the switch to ABCI++, at which point we can remove the `Vext` transactions entirely (since they will be inside ABCI++ proper vote extensions). So my thinking is it is safer to only have the code path active for the transactions we expect (`BridgePool`) and explicitly warn and noop if we somehow received a digest transaction (but like you say, it shouldn't make it this far in the code).

Reply:
Yeah, I guess this is an elegant way of handling it.